### PR TITLE
{lib}[foss/2021b] xTB v6.4.1

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/9d75cada706356578ac5bbb5068cd899 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/1eeb4e9aa5f666b9c1163a3a413609af for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/x/xTB/xTB-6.4.1-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+
+name = 'xTB'
```
[the remark this answers is not in the document]
Comment (Micket) [reply easybuild/easyconfigs/x/xTB/xTB-6.4.1-foss-2021b.eb:3]: I missed that one. Lets just drop this.

**Pull request closed without merging**